Sources/Rendering/Core/Mapper/index.js

```diff
@@ -53,6 +53,11 @@ function vtkMapper(publicAPI, model) {
     // make sure we do NOT call modified()
   };
 
+  publicAPI.setSelectionWebGLIdsToVTKIds = (selectionWebGLIdsToVTKIds) => {
+    model.selectionWebGLIdsToVTKIds = selectionWebGLIdsToVTKIds;
+    // make sure we do NOT call modified()
```

Comment:
Oh wait, I meant that you should add more context here, since someone reading the code might not know the reason for not calling modified().

Reply:
Oh sorry for misunderstanding your review!
I added the comment to the code